gap/elements/bipart.gi

```diff
@@ -93,6 +93,14 @@ InstallGlobalFunction(Bipartition,
 function(classes)
   local n, copy, i, j;
 
+  n := Sum(List(classes, Length)) / 2;
```

Comment:
In fact, `Sum(List(classes, Length))` can be replaced by simply `Sum(classes, Length)`.  I believe the same goes for `Product`.  Handy, eh?